doc/rst/developer/chips/10.rst

```diff
@@ -679,7 +679,8 @@ Deinitialization
 ++++++++++++++++
 
 To specify actions that occur upon deinitialization, specify a ``proc
-deinit()`` method. This method must have zero arguments and cannot be called
+deinit()`` method. This method must have zero arguments and be declared
```

Comment:
I would insert another "must" here, as in "This method must have zero arguments and must be declared with parentheses", though you certainly don't have to do so.